src/garage/tf/models/gaussian_conv_model.py

```diff
@@ -0,0 +1,301 @@
+                    hidden_w_init=self._hidden_w_init,
+                    hidden_b_init=self._hidden_b_init,
+                    output_nonlinearity=self._output_nonlinearity,
+                    output_w_init=self._output_w_init,
+                    output_b_init=self._output_b_init,
+                    name='mean_network',
+                    layer_normalization=self._layer_normalization)
+
+                # std network
+                if self._adaptive_std:
+                    log_std_conv = cnn(
+                        input_var=state_input,
+                        filter_dims=self._std_filter_dims,
+                        hidden_nonlinearity=self._std_hidden_nonlinearity,
+                        hidden_w_init=self._std_hidden_w_init,
+                        hidden_b_init=self._std_hidden_b_init,
+                        num_filters=self._std_num_filters,
+                        strides=self._std_strides,
+                        padding=self._std_padding,
+                        name='log_std_cnn')
+
+                    log_std_network = mlp(
+                        log_std_conv,
+                        output_dim=action_dim,
+                        hidden_sizes=self._std_hidden_sizes,
+                        hidden_nonlinearity=self._std_hidden_nonlinearity,
+                        hidden_w_init=self._std_hidden_w_init,
+                        hidden_b_init=self._std_hidden_b_init,
+                        output_nonlinearity=self._std_output_nonlinearity,
+                        output_w_init=self._std_output_w_init,
+                        output_b_init=tf.constant_initializer(
+                            self._init_std_param),
+                        name='log_std_network',
+                        layer_normalization=self._layer_normalization)
+                else:
+                    log_std_network = parameter(
+                        state_input,
+                        length=action_dim,
+                        initializer=tf.constant_initializer(
+                            self._init_std_param),
+                        trainable=self._learn_std,
+                        name='log_std_network',
+                        pixel_env=True)
+
+        mean_var = mean_network
+        std_param = log_std_network
+
+        with tf.variable_scope('std_limits'):
+            if self._min_std_param is not None:
+                std_param = tf.maximum(std_param, self._min_std_param)
+            if self._max_std_param is not None:
+                std_param = tf.minimum(std_param, self._max_std_param)
+
+        with tf.variable_scope('std_parameterization'):
+            # build std_var with std parameterization
+            if self._std_parameterization == 'exp':
+                log_std_var = std_param
+            else:  # we know it must be softplus here
+                log_std_var = tf.log(tf.log(1. + tf.exp(std_param)))
+
+        dist = DiagonalGaussian(self._output_dim)
```

Comment:
please also use a tensorflow_probability.distributions.MultivariateNormDiag, as in GaussianMLPModel

Reply:
I think currently we are still using `DiagonalGaussian` for `Gaussian*Model`.

Reply:
you are right. please ignore.